src/SearchField/index.jsx

```diff
@@ -10,21 +10,46 @@ import SearchFieldInput from './SearchFieldInput';
 import SearchFieldClearButton from './SearchFieldClearButton';
 import SearchFieldSubmitButton from './SearchFieldSubmitButton';
 
+const STYLE_VARIANTS = [
+  'light',
+  'dark',
+];
+
+const BUTTON_LOCATION_VARIANTS = [
```

Comment:
`STYLE_VARIANTS` and `BUTTON_LOCATION_VARIANTS` are defined several times. Would it make sense to export them from a constants.js file?

Reply:
I did this at the beginning but then in the Props API it displays variable name (e.g., `STYLE_VARIANTS`) instead of `'light' | 'dark'`. That is due to importing constants from another file. For now there is no better solution other than duplicating.